resources/common.js

```diff
@@ -343,7 +343,7 @@ $.fn.textWidth = function () {
 $(function () {
     $('.tabs').each(function () {
         var $this = $(this), $h2 = $(this).find('h2'), $ul = $(this).find('ul');
-        var cutoff = ($h2.textWidth() || 400) + 20, handler;
+        var cutoff = ($h2.textWidth() || 0) + 20, handler;
```

Comment:
what is this

Reply:
When the screen width is small, we need to push the second tab list to the next line, like this:

![image](https://user-images.githubusercontent.com/17219541/132246521-7782d8d9-628b-46b0-a87f-504273ba7929.png)

Originally, it assumes that there is an H2 heading followed by a tab list. When no H2 heading exists within the `tabs` div, it assumes there is one with a width of 400. This would break our org UI ([see](https://github.com/VNOI-Admin/OJ/pull/163/files#diff-0aa8ceea7c34106d322964ddbdb5e8c091b204123cc5a8cb426e380997a81bceR10-R31)) so I set it to 0. It shouldn't break anything.

Reply:
i see, legit